selfdrive/car/ford/carstate.py

```diff
@@ -15,6 +15,10 @@ def __init__(self, CP):
     super().__init__(CP)
     can_define = CANDefine(DBC[CP.carFingerprint]["pt"])
     if CP.transmissionType == TransmissionType.automatic:
+      # TODO
+      # - [x] NOPE: PowertrainData_10->GearLvr_D_ActlDrv (goes into L instead of D for the rest of the drive randomly, commonly when current message goes into unknown gear)
+      # - [ ] PowertrainData_10->TrnRng_D_Rq
+      # - [x] NOPE: TransGearData->GearLvrPos_D_Actl (goes into N instead of P, this is the transmission gear)
```

Comment:
The signal previously used before adding the Lincoln Aviator in https://github.com/commaai/openpilot/pull/27095 also goes to N around the time that these three candidates do, in cases where the current signal does not. It seems we are currently missing some cases where the user or car puts the gear into N which the camera listens to and disengages, so this seems like a good all around change.

![image](https://github.com/commaai/openpilot/assets/25857203/aae06b6d-42ba-47d8-a837-5e9abc8ced4d)